aopy/data/bmi3d.py

```diff
@@ -642,17 +643,22 @@ def get_interp_task_data(exp_data, exp_metadata, datatype='cursor', samplerate=1
 
     # Fetch the relevant BMI3D data
     if datatype == 'hand':
```

Comment:
hand == manual_input == user_raw

user_world

cursor == user_screen

user == intended_cursor = cursor - disturbance [this should equal user_world * get_world_to_screen_mapping()]
^ should print a warning if used, so "user" can be phased out

Reply:
Yeah I think that makes sense!

Reply:
hmm but it gets difficult when there are incremental mappings. do you have a function to apply the incremental mappings to the input data at the right times?

Reply:
Oh right. I don't think I've ever applied the incremental mappings to world coordinate data, I've just used the `user` datetype. To apply the incremental mappings to the right data, you'd have to know other exp params like `init_rotation_y`, `final_rotation_y`, `delta_rotation_y`, and `trials_per_increment`. But it's further complicated because you have to segment the data to know when new trials starts, when rewards happen, etc.